tests/hawc/apps/riskofbias/test_riskofbias_models.py

```diff
@@ -31,3 +32,34 @@ def test_get_orphan_relations(self):
             """
         ).strip()
         assert actual == expected
+
+
+@pytest.mark.django_db
+class TestRiskOfBias:
+    def test_get_dp_export(self):
+        assessment_id = 1
+        study_id = 1
+        _, scores_map = RiskOfBias.get_dp_export(assessment_id, [study_id], "animal")
+
+        # all of the metrics are mapped
+        set(
+            RiskOfBiasScore.objects.filter(
+                riskofbias__study=study_id, riskofbias__final=True, riskofbias__active=True,
+            ).values_list("metric_id")
+        ) == {metric for _, metric in scores_map.keys()}
+
+        # however, study has more scores than metrics
+        assert RiskOfBiasScore.objects.filter(
+            riskofbias__study=study_id, riskofbias__final=True, riskofbias__active=True,
+        ).count() > len(scores_map.keys())
+
+        # make sure only default scores are used
+        for (_, metric), score in scores_map.items():
```

Comment:
use .values() instead of items() since you're throwing away the key

Reply:
The key in `scores_map` is a tuple, so I'm only throwing away the first element from the tuple; I still need the second element, `metric`

Reply:
ahh, makes sense!